language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java

```diff
@@ -71,43 +71,53 @@
     private static final Logger logger = LoggerFactory.getLogger(CommandExecutor.class);
 
     private static final String ARG_KEY = "argumentK";
+
     private static final String ARG_VALUE = "argumentV";
+
     private static final String RUNTIME_PKG_ALIAS = ".runtime";
 
+    private CommandExecutor() {
+    }
+
     /**
      * Command Execution router.
      * @param params            Parameters for the command
      * @param context           Workspace service context
      */
-    public static void executeCommand(ExecuteCommandParams params, LSServiceOperationContext context) {
+    public static Object executeCommand(ExecuteCommandParams params, LSServiceOperationContext context) {
```

Comment:
We are returning the generic object here? Any reason for that, than specific type?

Reply:
@Kishanthan yes. Execute Command request is the language sever side's caller of this particular method. Command execution can for various commands will not return a specific type. That can be vary depending on the command type. Due to this reason WorkspaceService interface's executeCommand method also returns a generic object. 
https://github.com/ballerina-platform/ballerina-lang/blob/master/language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaWorkspaceService.java#L105

Reply:
Ok, but returning object is not good practice unless no other way. That is my concern. Btw, are using the return value some where?

Reply:
Yes, Usually this is used depending on what type of command, by the Language Clients such as VSCode and etc.